### DISCUSSION TITLE
Ankimon Next Update: Version 1.27

### DISCUSSION THREAD

**Discussion by Unlucky-Life** [Unlucky-Life/ankimon#52] [Announcements]
- PokeBall Icon for all windows (idea from @shigeyukey)
- better Font especially for japanese (big thanks to  @Foxy-null)
- Ankimon Window to center of Maxwindow on startup (this hopefully solves the fullscreen issue on mac)
- New Option: Having to press again can damage own mainpokemon. If this mainpokemon is defeated - the wild pokemon flees and mainpokemon is revived (Note: a more complex concept of damaging of own pokemon in reviewer will be incorporated for the future.)
- solved Linux issue with "license.html" file not found
- if a wild encounter pokemon is already caught in reviewer a pokeball is shown
- bug fix: new caught pokemon have attacks from pokemon before
- added missing back pokemon sprites
- more configuration options such as: xp bar in reviewer, remove and choose thickness of hp bar, different types of presentations of battle
- bug fix: on study, undo = card counter goes up by 1 with the help of @omfgroflmfaol
- bug fix: realtime refreshing of Hp bar
- can now heal mainpokemon with healing items
- can now evolve collected fossil items
- fixed aura issue
- sounds added ! (damaging, healing a pokemon etc. is now possible)
- small animation ! If a pokemon is being hit by a attack - you can let it animate!

**Comment by CaptainDead** [comment]
Perfect, I needed some of that stuff, great work💯🔥🙏

**Comment by Foxy-null** [comment]
Hey @Unlucky-Life, I ran into this bug while using the pre-release v1.269. I'm not sure why this error occurred, but I wanted to tell you about it anyway.

![image](https://github.com/Unlucky-Life/ankimon/assets/72304646/9537241c-0c93-44bd-a279-d47e478b7e26)
![image](https://github.com/Unlucky-Life/ankimon/assets/72304646/b3b78fbd-b102-4486-8976-97d2c32e7603)

**Reply by Unlucky-Life** [reply]
@Foxy-null hm this seems to be an issue with the enemy pokemon moves
How did that bug happen ?
I did reupload that prelease 2-3 times, so maybe that might be the issue,

**Reply by Foxy-null** [reply]
> @Foxy-null hm this seems to be an issue with the enemy pokemon moves
> 
> How did that bug happen ?
> 
> I did reupload that prelease 2-3 times, so maybe that might be the issue,

I only ran into this bug 2-3 times during my 8 hours of Anki studying yesterday. I'll try to update the add-on file to the latest version. If I encounter the bug again, I'll contact you with an enemy image.

**Reply by Unlucky-Life** [reply]
@Foxy-null hm weird - but it might just be moves that seem to be not found in the moves dictionary - ill check that out

**Reply by shigeyukey** [reply]
IMO, for code with occasional errors in game add-ons, I think it would be easy to set up actions or monsters for errors.
For example, in my custom Pokemanki, if an Error occurs and a Pokemon cannot be retrieved, it automatically becomes the Ditto.
This way cannot be used for regular add-ons, but for game add-ons do not need to be precise, so I do not think it's a problem.

**Reply by Unlucky-Life** [reply]
@shigeyukey Has the same idea back then, but didnt take care of that yet. But its integrated in the prerelease and will appear in the next update :) !
